### ISSUE TITLE
Problem with building pgRouting: PostgreSQL not found

### ISSUE THREAD

**Issue by lab348** [pgRouting/pgrouting#547]
### Expected behavior and actual behavior
After execution of command cmake .. in build directory I see such output. It looks that postgresql was found, but still in the end I see that "-- PostgreSQL not found" and build fail.
```
-- CMAKE_VERSION: 3.0.2
-- PGROUTING_GIT_TAG: pgrouting-2.2.0
-- PGROUTING_GIT_BUILD: RC1
-- PGROUTING_GIT_HASH: afc622e
-- PGROUTING_GIT_BRANCH: master
-- PGROUTING_VERSION_REVISION_NAME: afc622e master
-- PGROUTING_SOURCE_DIR: /root/pgrouting-pgrouting-2.2.0
-- CMAKE_BINARY_DIR: /root/pgrouting-pgrouting-2.2.0/build
-- PGROUTING_VERSION_REVISION_HASH: afc622e
-- POSTGRESQL_BIN: 
-- UNIX=1
-- WIN32=
-- POSTGRESQL_PG_CONFIG is /usr/bin/pg_config
-- POSTGRESQL_EXECUTABLE is /usr/lib/postgresql/9.4/bin/postgres
-- POSTGRESQL_VERSION_STRING in FindPostgreSQL.cmake is PostgreSQL 9.4.6
-- PostgreSQL not found.
CMake Error at CMakeLists.txt:113 (message):
   Please check your PostgreSQL installation.


-- Configuring incomplete, errors occurred!
See also "/root/pgrouting-pgrouting-2.2.0/build/CMakeFiles/CMakeOutput.log".
```

### Steps to reproduce the problem
Execute cmake.. in build directory.

### Specifications like the version of pgRouting/PostGIS and PostgreSQL as well as Operating System
- Debian 8.3
- pgRouting 2.2.0 -> https://github.com/pgRouting/pgrouting/archive/pgrouting-2.2.0.zip
- PostgreSQL 9.4.6 installed as a package

**Comment by cvvergara** [comment]
maybe you are missing postgresql server:
http://docs.pgrouting.org/2.2/en/doc/src/installation/install-dependencies.html#postgresql-version-9-1

**Comment by robe2** [comment]
Did you install the PostgreSQL dev package?  

```
sudo apt-get install postgresql-server-dev-9.4
```

That's needed to compile any PostgreSQL extension.

**Comment by lab348** [comment]
You are right. This was the missing thing. Now everything is ok. Thanks.

**Comment by zsc1220** [comment]
hello，I also meet this problem now, Could you tell me how did you solve this problem,thank you very much!

**Comment by dkastl** [comment]
@zsc1220 , have you read the answer above? This was the solution.

**Comment by zsc1220** [comment]
There are two methods above, and he doesn't say which one is right, I tried ,but not work

**Comment by cvvergara** [comment]
Version 2.2.2 is old, from Apr 28, 2016
We are on version 2.6 now.
Please try with version 2.6.1 (not even try with 2.6.0, because it has an serious bug issue fixed on 2.6.1)
Please keep us posted on what happens when you try it.

Also by the time of the comment https://github.com/pgRouting/pgrouting/issues/547#issuecomment-214006119, postgres 9.4 was being developed, so this is also outdated:
`sudo apt-get install postgresql-server-dev-9.4`
